ocs_ci/ocs/node.py

```diff
@@ -8,7 +8,7 @@
 from ocs_ci.ocs import constants, exceptions
 from ocs_ci.utility.utils import TimeoutSampler
 from ocs_ci.ocs import machine
-from tests.helpers import get_worker_nodes
+import tests.helpers
```

Comment:
Again, to prevent circular import error. `from tests.helpers import get_worker_nodes` executes tests.helpers here and now, `import tests.helpers` defers execution.